docs/vault/guide.md

```diff
@@ -113,11 +113,69 @@ To mint with your own (or another specific) Vault, you can use the Polkadot.js i
 
 You Vault only starts earning rewards once BTC is locked - and rewards are determined by your share of the total BTC locked in the system. Hence, you can increase your rewards by bringing your own BTC into the system. 
 
-## Dashboard
+## Monitoring
```

Comment:
Please adopt the guide to be similar to https://wiki.polkadot.network/docs/maintain-guides-how-to-monitor-your-node#installing-and-configuring-alertmanager-optional in the detail level.

Most important questions this guide should answer are:
- How do vaults know if their client is going offline?
- How do vaults know if their collateralization is getting close to the liquidation threshold?
- How do vaults know if they need to check their clients if redeem requests are close to timeout?

Reply:
> How do vaults know if their collateralization is getting close to the liquidation threshold?
Good point, ideally the monitoring module should query the thresholds from the chain and have those displayed in Grafana. For now, operators will just have to be aware of the thresholds. I'll add a ticket for this improvement.

> How do vaults know if they need to check their clients if redeem requests are close to timeout?
This is also not available in the MVP. A Grafana tile for this would show the oldest open redeem request and the cancellation period. Will add a ticket.

Reply:
Thanks! I think that would then be the next highest priority items for the metrics. cc @sander2 @gregdhill